packages/polymath-issuer/src/pages/compliance/components/EditApprovalForm.js

```diff
@@ -0,0 +1,250 @@
+// @flow
+
```

Comment:
Out of curiosity, could we've used one form component for both adding and editing, white disabling addresses fields in the later case?

Reply:
I did this last time for VRTM and it became such a mess it was better to have them a separated concerns.